app/com/m3/octoparts/ZipkinServiceHolder.scala

```diff
@@ -13,6 +14,10 @@ object ZipkinServiceHolder {
   private implicit val app = play.api.Play.current
   private implicit val ex = play.api.libs.concurrent.Execution.Implicits.defaultContext
 
+  def spanNamer(r: RequestHeader): String = {
+    s"${r.method} - ${r.path.replaceAll("/[0-9]+", "/XXX")}"
```

Comment:
This solves our use case, but because the $paramValue in cache invalidation API does not have to be numeric, does not work in the general case.